snapcraft/project/errors.py

```diff
@@ -58,3 +60,31 @@ def get_brief(self) -> str:
 
     def get_resolution(self) -> str:
         return "This extension may be enabled with the '--enable-experimental-extensions' parameter."
+
+
+class UnsupportedBase(SnapcraftException):
```

Comment:
for consistency, perhaps
```suggestion
class UnsupportedBaseError(SnapcraftException):
```